webpack/ForemanTasks/Components/TasksTable/__tests__/__snapshots__/TasksTableReducer.test.js.snap

```diff
@@ -20,6 +20,7 @@ Object {
 exports[`TasksTableReducer reducer should handle TASKS_TABLE_SELECTED_MODAL 1`] = `
 Object {
   "tasksTableQuery": Object {
+    "clicked": undefined,
```

Comment:
What is this `clicked` for? Should an initial value be set so it's not undefined?

Reply:
It is the details for the task the you just clicked on to cancel/resume, since no task is clicked it is undefined